frontend/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const SCROLL_THRESHOLD = 300 // pixels to scroll before showing the scroll-up button
```

Comment:
We already have `constants` file in `utils`. Maybe you could move this into that file? 
And then update imports, since right now it fails on load:
![Screenshot 2025-02-23 at 11 50 37 AM](https://github.com/user-attachments/assets/7638c10e-530b-446a-b642-516ce8304f35)
![Screenshot 2025-02-23 at 11 51 25 AM](https://github.com/user-attachments/assets/a88c935a-7925-44f6-8a37-ec5b955faafc)

Reply:
@iitzIrFan when you start server locally this should also pop up in your terminal 🤔 Were you able to run the project locally?